vhdbuilder/packer/install-dependencies.sh

```diff
@@ -449,11 +466,23 @@ rm -r /var/log/azure/Microsoft.Azure.Extensions.CustomScript || exit 1
 
 KUBE_PROXY_IMAGE_VERSIONS=$(jq -r '.containerdKubeProxyImages.ContainerImages[0].multiArchVersions[]' <"$THIS_DIR/kube-proxy-images.json")
 
+declare -a kube_proxy_pids=()
+
+for KUBE_PROXY_IMAGE_VERSION in ${KUBE_PROXY_IMAGE_VERSIONS}; do
+  # use kube-proxy as well
```

Comment:
remove this comment?

Reply:
Haha I felt the same way! It has been there for a while I did not make it, I considered removing but wasn't sure why it was there so I didn't

Reply:
I can remove if you would like, from both loops

Reply:
That'll be neat, thanks!